src/core_modules/capture-core/components/Pages/MainPage/OfflineEventsList/OfflineEventsList.container.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import { connect } from 'react-redux';
 import OfflineEventsList from '../../../EventsList/OfflineEventsList/OfflineEventsList.component';
-import { openEditEventPage } from '../EventsList/eventsList.actions';
+import { openEditEventPage } from '../../../ListView/listView.actions';
```

Comment:
Do we maybe want to export the `openEditEventPage` from the index of the `ListView`?

Reply:
Removed in later PR.